python/tk_multi_publish2/dialog.py

```diff
@@ -258,12 +260,24 @@ def _on_item_comment_change(self):
         """
         comments = self.ui.item_comments.toPlainText()
         if self._current_item is None:
-            # this is the summary item - so update all items!
-            for top_level_item in self._plugin_manager.top_level_items:
```

Comment:
we want to update all items only if the summary comment has changed... so added a check for that